cli/cmd/deploy/deploy.go

```diff
@@ -282,6 +275,10 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 
 			// Success!
 			success.Printf("Created project \"%s/%s\". Use `rill project rename` to change name if required.\n\n", cfg.Org, res.Project.Name)
+			// Run flow to check connector credentials
+			if err := variablesFlow(ctx, fullProjectPath, name); err != nil {
+				return err
+			}
 			success.Printf("Rill projects deploy continuously when you push changes to Github.\n")
```

Comment:
1. Move the log message up before the variables flow
2. If the variables flow fails, probably just log the error instead of returning/exiting with it